src/github3/repos/repo.py

```diff
@@ -600,6 +602,8 @@ def clones(self, per='day'):
         params = {}
         if per in ("day", "week"):
             params.update(per=per)
+        else:
+            raise ValueError("per must be 'day' or 'week'")
```

Comment:
This looks like the right error to raise. But when adding a Raise we should update the function doc string. I was just looking at [this example](https://github.com/sigmavirus24/github3.py/blob/master/src/github3/orgs.py#L525-L551) and saw the docs were updated there.

Reply:
Of course, thanks for catching that.